.devcontainer/post-create.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ "${CODESPACES}" == true ]]; then
+  echo "Fixing permissions of /tmp for GitHub Codespaces..." >&2
+  sudo chmod 1777 /tmp
```

Comment:
I don't think we've sudo available on that image

Reply:
We have, thanks to the common-utils feature: https://github.com/devcontainers/features/tree/main/src/common-utils#readme

Reply:
ok.